duckduckgo/macos-browser
# Update copy

Task/Issue URL:https://app.asana.com/0/1204099484721401/1205190239394744

**Description**:
- Updates the copy for fireproof site message

**Steps to test this PR**:
1. Go to fill.dev
2. Enter random user and pass (do not auto-generate)
3. Check the copy in the dialog matches this screenshot

<img width="362" alt="Screenshot 2023-08-21 at 10 49 39 AM" src="https://github.com/duckduckgo/macos-browser/assets/1156669/3ca38c26-8903-40ef-81d8-b0b286131f49">



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
